README.md

```diff
@@ -154,6 +154,33 @@ yarn run dev
 ```
 If changes aren't detected, you might need to increase `fs.inotify.max_user_watches`. See <https://webpack.js.org/configuration/watch/#not-enough-watchers>.
 
+#### Pushing Changes to a Remote Cluster
+
+Once you have made changes locally, these instructions will allow you to push changes to a remote Cluster for others to review.  This involves building a local image, pushing the image to Quay, then updating a remote cluster to pull the new image.
+
+##### Prerequisites
+
+1. Docker v17.05 or higher
+2. Quay Account <br> https://quay.io/signin/
+    - Sign in using GitHub account
+    - Logging into Quay from the Docker CLI
+      - Quay Account Settings - > Generate CLI Password: Generate Encrypted Password
+      ```
+      sudo docker login quay.io
+      Username: userFooBar
+      Password: ( password here)```
+    *Highly recommend going through Quay Tutorial which walks you though logging in with Docker and pushing a test image.
+##### Steps
+1. Create a Quay repository <br> Login to https://quay.io, and create a blank, empty repository.  For this example, we'll use _devchanges_ as our example repo; created under _userFooBar_ quay account.
+2. Build Image <br> `sudo docker build -t devchanges github.com/userFooBar/devchanges`
+3. Push to quay <br> `sudo docker push quay.io/userFooBar/devchanges`
+4. Put the console operator in unmanaged state <br> `oc edit consoles.operator.openshift.io cluster` <br> set `spec: managementState: Unmanaged` and save.  (vi -> ‘[i]nsert’, `Unmanaged`, ‘:wq’)'
```

Comment:
I would skip the vi commands. Let's assume the user knows their default editor. I'd show this as a multiline example to avoid confusion, like

```yaml
spec:
  managementState: Unmanaged
```